multihost/db-postgresql-with-custom-certificates/main.fmf

```diff
@@ -0,0 +1,38 @@
+summary: Test basic keylime attestation scenario on multiple hosts with custom certificates and postgresql database
+description: |
+ Running services on different systems - verifier + registrar + tenant ( verifier and registrar configured using a keylime_server role ), agent, setup server.
```

Comment:
FYI, what you call setup server is being called ansible controller. Maybe we could use controller as a role as well.

Reply:
Agree, I'll update it.